src/main/java/org/openstreetmap/atlas/geography/boundary/CountryBoundaryMap.java

```diff
@@ -363,7 +363,7 @@ public static void setGeometryProperty(final Geometry geometry, final String key
                 // Trying to override an existing value - this shouldn't happen!
                 if (!Objects.equals(existingValue, value))
```

Comment:
How about this, so the geometry is not computed? Or is it going too far?
```suggestion
                if (!Objects.equals(existingValue, value) && logger.isDebugEnabled())
```